platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -921,6 +1270,7 @@ namespace platoon_strategic_ihp
         // Check joining plan type.
         bool isFrontJoin = (plan_type.type == cav_msgs::PlanType::JOIN_PLATOON_FROM_FRONT);
         bool isRearJoin = (plan_type.type == cav_msgs::PlanType::JOIN_PLATOON_AT_REAR);
+        bool isCutinJoin = (plan_type.type == cav_msgs::PlanType::CUT_IN_FROM_SIDE);
```

Comment:
Presumably all cut-ins are from the side.  So setting this could still indicate a front or rear join (from adjacent lane), right?